src/components/User.stories.js

```diff
@@ -0,0 +1,13 @@
+import User from './User'
+export default {
+  title: 'User',
+  component: User,
+}
+
+const Template = args => <User {...args} />
```

Comment:
I would think that you don't have props (as I said before) but maybe I get it wrong the whole time